pkg/kf/commands/utils/prefix_writer.go

```diff
@@ -0,0 +1,71 @@
+package utils
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+)
+
+type prefixWriter struct {
+	nw newlineWriter
+}
+
+func NewPrefixWriter(prefix string, dest io.Writer) io.Writer {
```

Comment:
I'd recommend using something off the shelf like https://github.com/segmentio/textio rather than DIYing this

Reply:
I thought about that, but its simple enough I figured I would rather avoid the dependency. Thoughts?

Reply:
I like that it doesn't bring in any dependencies and seems to be well written but I can see the desire to keep our dependencies slim.